packages/dd-trace/src/appsec/iast/analyzers/cookie-analyzer.js

```diff
@@ -11,7 +11,9 @@ class CookieAnalyzer extends Analyzer {
     this.propertyToBeSafe = propertyToBeSafe.toLowerCase()
   }
 
-  onConfigure () {
+  onConfigure (config) {
+    this.cookieFilterRegExp = new RegExp(config.iast.cookieFilterPattern)
```

Comment:
This could throw a `SyntaxError` that, if not managed, will lead to IAST to not be enabled. I think we should catch it here and manage it.